nss-ivory-trolls/BangazonSite
# Add Payment Type - Ticket #7

# Description

**Given** the user is authenticated
**When** the user clicks on a profile affordance in the navigation bar
**And** the user clicks on the Add Payment Option hyperlink in their profile
**Then** the user can fill out the payment option form
**And** click Submit to save payment option


## Type of change


# Testing Instructions for Change Made

```
git fetch -a
git checkout ab-addptype
```

1. Log into an account
2. Click on the username in the nav bar
3. Click "Add Payment Type" link on the account management page
4. Add a payment type
5. Confirm payment type was added in the index and assigned to the current user


# Checklist:
